orchagent/fabricportsorch.cpp

```diff
@@ -24,6 +24,9 @@
 #define FABRIC_DEBUG_POLLING_INTERVAL_DEFAULT   (60)
 #define FABRIC_MONITOR_DATA "FABRIC_MONITOR_DATA"
 #define APPL_FABRIC_PORT_PREFIX "Fabric"
+#define SWITCH_DEBUG_COUNTER_FLEX_COUNTER_GROUP  "SWITCH_DEBUG_COUNTER"
+#define SWITCH_DEBUG_COUNTER_POLLING_INTERVAL_MS 60000
```

Comment:
@saksarav-nokia is this 60 secs ? Any reason we are picking 60 secs as polling ?